lib/Logger/LoggerFeature.h

```diff
@@ -70,7 +70,8 @@ class LoggerFeature final : public application_features::ApplicationFeature {
   bool _useJson = false;
   bool _useLocalTime = false;
   bool _useColor = true;
-  bool _useEscaped = true;
+  bool _useControlEscaped = true;
+  bool _useUnicodeEscaped = true;
```

Comment:
Shouldn't this flag be `false` by default?
```suggestion
  bool _useUnicodeEscaped = false;
```